dev-env/update-openapi

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
In other scripts, such as "run-docker-compose.sh", we are using
```bash
SCRIPT_DIR="$(realpath -e "$(dirname "$0")")"
```
which seems more readable. Is it equivalent?

Reply:
There are a lot of comments on https://stackoverflow.com/questions/59895/how-can-i-get-the-source-directory-of-a-bash-script-from-within-the-script-itsel. I took the most upvoted solution while still being a one-liner. It seems to handle more cases, but we probably don't need to handle all these situations so the simpler one is probably enough.

Apparently using `${BASH_SOURCE[0]}` instead of `$0` makes it work if the script is evaluated with `source my_script`, and `realpath` is not always available (requires coreutils on macOS).

I didn't think to look at the other scripts. I'll copy them for consistency.